distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -18,9 +18,41 @@ Adding other components does not have any security/hardening implications.
 
 These are the system/environment requirements to configure _prior_ to an installation.
 
-.Hardware Requirements
-* At least 4096MB of memory for ${branding}.
-** This amount can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>.
+====== Minimum and Recommended Requirements for ${branding} Systems in Production Mode
```

Comment:
I originally had `Hardware` in the name, but dropped it since we also reference browser, jdk and OS. Will add it back.